### PR TITLE
Fix nightly build

### PR DESCRIPTION
**Pull request by RafalKorepta**
```
ct lint --all --config .github/ct-redpanda.yaml
Linting charts...
Error: failed loading configuration: specifying both, '--all' and '--charts', is not allowed
failed loading configuration: specifying both, '--all' and '--charts', is not allowed
Error: Process completed with exit code 1.
```

https://github.com/redpanda-data/helm-charts/actions/runs/5687162122/job/15415137752#step:6:11

### CODE REVIEW

File: .github/workflows/nightly.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
           version: v3.8.0-1
 
       - name: Run chart-testing (lint)
-        run: ct lint --all --config .github/ct-redpanda.yaml
+        run: ct lint --config .github/ct.yaml
```
Comment (joejulian) [inline .github/workflows/nightly.yaml:46]: Don't we still need
```suggestion
        run: ct lint --config .github/ct-redpanda.yaml
```

Comment (RafalKorepta) [reply .github/workflows/nightly.yaml:46]: I was thinking that nightly should lint all charts. That's why I bring back ct.yaml configuration. WDYT?

Comment (alejandroEsc) [reply .github/workflows/nightly.yaml:46]: What is the intent of the nightly now? I thought it was to run tests only for the redpanda chart, after all we have a matrix. Should we have a nightly for all charts that just checks linting?

Comment (RafalKorepta) [reply .github/workflows/nightly.yaml]: reverted

**Comment by RafalKorepta** [comment]
I pushed one correction to this fix as none of the Github action workflow was triggered. Our PR requires `summary` and `lint` jobs.

**Pull request merged**